device/bgp/evpn/bgpevpncmacranges.yaml

```diff
@@ -0,0 +1,62 @@
+components:
+  schemas:
+    Bgp.CMacIpRange:
```

Comment:
It will better to expose name which will referred at the time of flow tx_rx configuration.